### PR TITLE
cnf-tests: add PSA labels to the default namespace

### PR DESCRIPTION
**Pull request by SchSeba**
This is needed because the sctp tests use the default namespace

Signed-off-by: Sebastian Sch <sebassch@gmail.com>

### CODE REVIEW

File: cnf-tests/testsuites/pkg/features/features.go [hunk 1]
```diff
@@ -110,6 +109,12 @@ type SCTPFixture struct {
 }
 
 func (p *SCTPFixture) Setup() error {
+	err := namespaces.AddPSALabelsToNamespace(namespaces.SCTPTest, testclient.Client)
+	if err != nil {
+		return err
+	}
+
+	err = namespaces.AddPSALabelsToNamespace(namespaces.Default, testclient.Client)
```
Comment (zeeke) [inline cnf-tests/testsuites/pkg/features/features.go:117]: `err` variable is not  used I think.

File: cnf-tests/testsuites/pkg/namespaces/namespaces.go [hunk 2]
```diff
@@ -131,6 +131,24 @@ func Create(namespace string, cs corev1client.NamespacesGetter) error {
...
+	if ns.Labels == nil {
+		ns.Labels = GetPSALabels()
+		return nil
+	}
+
+	for k, v := range GetPSALabels() {
+		ns.Labels[k] = v
+	}
+
+	return nil
```
Comment (zeeke) [inline cnf-tests/testsuites/pkg/namespaces/namespaces.go:149]: This function has no effect. Maybe it is missing an Update on cluster?

```suggestion
    _, err = cs.Namespaces().Update(context.Background(), ns, metav1.UpdateOptions{})
    return err
```

**Comment by SchSeba** [comment]
Thanks @zeeke !

**Comment by SchSeba** [comment]
gatekeeper will be disabled in https://github.com/openshift-kni/cnf-features-deploy/pull/1260
and the PAO errors are not related to this PR

/override ci/prow/e2e-gcp-ovn

**Pull request merged**